provider/haproxy/config/rsyslogd.conf

```diff
@@ -3,15 +3,12 @@ $UDPServerRun 8514
 $template CustomFormat,"%timegenerated% %HOSTNAME% %syslogtag%%msg%\n"
 $ActionFileDefaultTemplate CustomFormat
 
-/* info */
-if $programname contains 'haproxy' and $syslogseverity == 6 then
-    action(type="omfile" file="/var/log/haproxy/traffic")
-
-if $programname contains 'haproxy' and $syslogseverity-text == 'err' then
-    action(type="omfile" file="/var/log/haproxy/errors")
-
-/* notice */
-if $programname contains 'haproxy' and $syslogseverity <= 5 then
-    action(type="omfile" file="/var/log/haproxy/events")
-
-*.* /var/log/messages
+if $programname contains 'haproxy' then {
+  if $syslogseverity == 6 then
+      action(type="omfile" file="/var/log/haproxy/traffic")
+  if $syslogseverity-text == 'err' then
```

Comment:
Overlapping will help in easier debugging, but we'll now be redirecting these logs to show up in UI.
That might create confusion if errors show up in events section as well?